.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 💬 Help & Questions
+    url: https://github.com/sequelize/website/discussions
```

```suggestion
    url: https://github.com/sequelize/sequelize/discussions
```
Let's keep all discussions in one place